code/game/asteroid.dm

```diff
@@ -177,7 +177,7 @@ var/global/max_secret_rooms = 6
 			valid = 0
 			continue
 
-		if(locate(/turf/simulated/floor/plating/airless/asteroid) in range(5,T))//A little less strict than the other checks due to tunnels
+		if(locate(/turf/simulated/floor/plating/asteroid/airless) in range(5,T))//A little less strict than the other checks due to tunnels

```

Comment:
whoever thought `airless/asteroid` was a good idea, instead of `asteroid/airless` should probably be pushed into a lavaland chasm.